tests/pipeline/test_blaze.py

```diff
@@ -1377,7 +1393,87 @@ def test_deltas(self, asset_info, add_extra_sid):
                 compute_fn=np.nanmax,
             )
 
-    @with_extra_sid
+    @with_ignore_sid()
+    def test_deltas_before_index_0(self, asset_info, add_extra_sid):
+        df = pd.DataFrame(np.array(
+            [],
+            dtype=[
+                ('sid', 'int64'),
+                ('value', 'float64'),
+                ('asof_date', 'datetime64[ns]'),
+                ('timestamp', 'datetime64[ns]'),
+            ],
+        ))
+        expr = bz.data(df, name='expr', dshape=self.dshape)
+
+        T = pd.Timestamp
+        deltas_df_single_sid = pd.DataFrame({
```

Comment:
This made sense to me after you explained it in person, but I wouldn't have figured it out easily by myself.  A couple sentences explaining what's interesting about the data under test here would be helpful.